src/report.js

```diff
@@ -1,21 +1,17 @@
-export const generateReport = async (results) => {
-    const lines = [];
-
-    if (results.length === 0) {
-        lines.push(`No files to generate SEO content were detected in this pull request.`);
-        lines.push(`-- [Jekyll SEO AI Action](https://github.com/AlphaGit/jekyll-seo-ai-action)`);
+    export const generateReport = (results) => {
+        const lines = [];
+        if (results.length === 0) {
+            lines.push(`No files to generate SEO content were detected in this pull request.`);
+            lines.push(`-- [Jekyll SEO AI Action](https://github.com/AlphaGit/jekyll-seo-ai-action)`);
+        } else {
+            lines.push(``);
```

Comment:
In this case of the file report, add a line with the markdown title inside:

lines.push(`# Updated files`);